cli/fix/fix.go

```diff
@@ -50,13 +51,18 @@ func walk() error {
 				return nil
 			}
 			fileName := filepath.Base(path)
-			fileExt := filepath.Ext(fileName)
-			fileNameRoot := strings.TrimSuffix(fileName, fileExt)
-			if (fileNameRoot != "BUILD" && fileNameRoot != "WORKSPACE") ||
-				(fileExt != "" && fileExt != ".bazel") {
+			fileNameRoot := strings.TrimSuffix(fileName, filepath.Ext(fileName))
+			if fileNameRoot != "BUILD" && fileNameRoot != "WORKSPACE" {
 				return nil
 			}
-			runBuildifier(path)
+			fileToFormat, err := translate.Translate(path)
+			if err != nil {
+				return err
+			}
+			if fileToFormat == "" {
+				return nil
+			}
```

Comment:
does runBuildifier still get hit if no translation is done?

Reply:
Only if it's a BUILD or WORKSPACE file - otherwise `fileToFormat` is set to the empty string.